Applications/CMC/test/testCMCWithControlConstraintsGait2354.cpp

```diff
@@ -65,28 +65,27 @@ void testGait2354() {
 
     int nq = results.getColumnLabels().getSize()-1;
 
-    // Tracking kinematics angles in degrees should be within 2 degrees
-    std::vector<double> rms_tols(nq, 2.0);
+    // Tracking kinematics angles in degrees should be within 1.5 degrees
+    std::vector<double> rms_tols(nq, 1.5);
     rms_tols[3] = 0.005; // pelvis translations in m should be with 5mm
     rms_tols[4] = 0.005;
     rms_tols[5] = 0.005;
 
-    CHECK_STORAGE_AGAINST_STANDARD(results, standard, rms_tols, 
+    CHECK_STORAGE_AGAINST_STANDARD(results, standard, rms_tols,
         __FILE__, __LINE__, "testGait2354 tracking failed");
 
     Storage results2("subject01_ResultsCMC/subject01_walk1_states.sto");
     Storage standard2("std_subject01_walk1_states.sto");
 
     Array<string> col_labels = standard2.getColumnLabels();
-    std::vector<double> rms_tols2(col_labels.getSize()-1, 0.1);
-    for (int i = 23; i < 46; ++i){
-        rms_tols2[i] = 0.75; // velocities
-    }
-    for (size_t i = 46; i < rms_tols2.size(); ++i){
-        rms_tols2[i] = 0.15; // muscle activations and fiber-lengths
+
+    // tolerance for joint angles 0.02rad ~= 1.15 degs and activation within 2%
+    std::vector<double> rms_tols2(col_labels.getSize()-1, 0.02);
+    for (int i = 0; i < nq; ++i){
+        rms_tols2[2*i+1] = 0.25; // velocities rad/s
```

Comment:
I agree. But that wasn't the reason for the change. Tightening the tolerances was the main purpose. As far as I can tell the order has been wrong since we started using the `MultibodyGraphMaker` but the tolerances were to loose make a difference.